packages/block-library/src/home-link/index.php

```diff
@@ -0,0 +1,178 @@
+		$colors['inline_styles'] .= sprintf( 'color: %s;', $context['style']['color']['text'] );
+	}
+
+	// Background color.
+	$has_named_background_color  = array_key_exists( 'backgroundColor', $context );
+	$has_custom_background_color = isset( $context['style']['color']['background'] );
+
+	// If has background color.
+	if ( $has_custom_background_color || $has_named_background_color ) {
+		// Add has-background class.
+		$colors['css_classes'][] = 'has-background';
+	}
+
+	if ( $has_named_background_color ) {
+		// Add the background-color class.
+		$colors['css_classes'][] = sprintf( 'has-%s-background-color', $context['backgroundColor'] );
+	} elseif ( $has_custom_background_color ) {
+		// Add the custom background-color inline style.
+		$colors['inline_styles'] .= sprintf( 'background-color: %s;', $context['style']['color']['background'] );
+	}
+
+	return $colors;
+}
+
+/**
+ * Build an array with CSS classes and inline styles defining the font sizes
+ * which will be applied to the home link markup in the front-end.
+ *
+ * @param  array $context Home link block context.
+ * @return array Font size CSS classes and inline styles.
+ */
+function block_core_home_link_build_css_font_sizes( $context ) {
+	// CSS classes.
+	$font_sizes = array(
+		'css_classes'   => array(),
+		'inline_styles' => '',
+	);
+
+	$has_named_font_size  = array_key_exists( 'fontSize', $context );
+	$has_custom_font_size = isset( $context['style']['typography']['fontSize'] );
+
+	if ( $has_named_font_size ) {
+		// Add the font size class.
+		$font_sizes['css_classes'][] = sprintf( 'has-%s-font-size', $context['fontSize'] );
+	} elseif ( $has_custom_font_size ) {
+		// Add the custom font size inline style.
+		$font_sizes['inline_styles'] = sprintf( 'font-size: %spx;', $context['style']['typography']['fontSize'] );
+	}
+
+	return $font_sizes;
+}
+
+/**
+ * Builds an array with classes and style for the li wrapper
+ *
+ * @param  array $context    Home link block context.
+ * @return array The li wrapper attributes.
+ */
+function block_core_home_link_build_li_wrapper_attributes( $context ) {
+	$colors          = block_core_home_link_build_css_colors( $context );
+	$font_sizes      = block_core_home_link_build_css_font_sizes( $context );
+	$classes         = array_merge(
+		$colors['css_classes'],
+		$font_sizes['css_classes']
+	);
+	$style_attribute = ( $colors['inline_styles'] . $font_sizes['inline_styles'] );
+	$css_classes     = trim( implode( ' ', $classes ) );
+
+	$wrapper_attributes = get_block_wrapper_attributes(
+		array(
+			'class' => $css_classes,
+			'style' => $style_attribute,
+		)
+	);
+
+	return $wrapper_attributes;
+}
+
+/**
+ * Renders the `core/home-link` block.
+ *
+ * @param array $attributes The block attributes.
+ * @param array $content    The saved content.
+ * @param array $block      The parsed block.
+ *
+ * @return string Returns the post content with the home url added.
+ */
+function render_block_core_home( $attributes, $content, $block ) {
```

Comment:
```suggestion
function render_block_core_home_link( $attributes, $content, $block ) {
```

Reply:
Thanks @swissspidy, I missed these in the home -> home_link rename. I've updated these in 10b9f65ab13fafad3794f30baefa5e147d6c3a9a